keyboards/qaz/config.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x7431
+#define PRODUCT_ID 0x0001
+#define DEVICE_VER 0x0001
+#define MANUFACTURER whydobearsexplod
+#define PRODUCT qaz keyboard
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 7
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS { F4, D4, C6, E6, D1, D0 }
+#define MATRIX_COL_PINS { B4, D3, D2, F5, B5, F6, D7 }
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+#define RGB_DI_PIN F7
+
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
+#define RGBLED_NUM 21
+#define RGBLIGHT_HUE_STEP 8
+#define RGBLIGHT_SAT_STEP 8
+#define RGBLIGHT_VAL_STEP 8
+#endif
+
+#ifdef COMBO_ENABLE
+   #define COMBO_COUNT 5
+   #define COMBO_TERM 200
+#endif
```

Comment:
Should be moved to a keymap level `config.h`
```suggestion
```

Reply:
What is the reason for this? The default keymaps do use combos. I can take out the #ifdef. Is there a convention I'm unaware of?

Reply:
The Configurator doesn't use the default keymap, it generates its own keymap.c. Because the Combo feature requires custom code, it will fail to build if it's enabled at the keyboard level.

Reply:
Ah ok no prob! will move